lib/shopping_cart.rb

```diff
@@ -0,0 +1,60 @@
+class ShoppingCart
+  attr_reader :name, :capacity, :products
+  attr_accessor :products_in_cart
+
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+    @products = []
+    @products_in_cart = 0
```

Comment:
I think this should not be an instance variable, as the `products_in_cart` is dynamically calculated based off of the `@products`.